cmd/kubeadm/app/phases/kubelet/flags.go

```diff
@@ -50,15 +51,22 @@ func buildKubeletArgMap(cfg *kubeadmapi.MasterConfiguration) map[string]string {
 		kubeletFlags["network-plugin"] = "cni"
 		kubeletFlags["cni-conf-dir"] = "/etc/cni/net.d"
 		kubeletFlags["cni-bin-dir"] = "/opt/cni/bin"
+		driver, err := kubeadmutil.GetCgroupDriverDocker()
+		if err != nil {
+			glog.Warningf("cannot assign a value for '--cgroup-driver': %v\n", err)
```

Comment:
This error message is confusing, you're just getting the cgroup-driver here, shouldn't that be an error?

Reply:
should it be made more detailed in the lines of?
`cannot automatically assign a '--cgroup-driver' value when starting the Kubelet: %v\n", err`

if `docker info` fails i wanted this to be a non fatal error.

Reply:
> cannot automatically assign a '--cgroup-driver' value when starting the Kubelet: %v\n", err

@neolit123 that sounds good